internal/csi-addons/networkfence/fencing.go

```diff
@@ -38,6 +39,27 @@ type NetworkFence struct {
 	cr       *util.Credentials
 }
 
+// ActiveClient represents the structure of an active client.
+type ActiveClient struct {
```

Comment:
this struct is not used outside of this package, you do not need to export it, so call it `activeClient`.